supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiBackendLifecycleActorFactory.scala

```diff
@@ -111,27 +108,26 @@ object PipelinesApiBackendLifecycleActorFactory {
       t match {
         case e: Exception =>
           Option(e.getMessage) match {
-            case Some(message) => message.contains("We encountered an internal error. Please try again.")
+            case Some(message) => message.contains("Please try again.")
             case None => false
           }
         case _ => false
       }
     }
 
     // `attempt` is 1-based
-    @tailrec
-    def build(attempt: Int): PipelinesApiConfigurationAttributes = {
+    def build(attempt: Int): Try[PipelinesApiConfigurationAttributes] = {
       Try {
         buildAttributes()
-      } match {
-        case Success(value) => value
-        case Failure(t) if isRetryableException(t) && attempt < maxAttempts =>
+      } recoverWith {
+        case t if isRetryableException(t) && attempt < maxAttempts =>
           logger.warn(s"Failed to build PipelinesApiConfigurationAttributes on attempt $attempt of $maxAttempts, retrying.", t)
           Thread.sleep(backoff.nextBackOffMillis())
```

Comment:
Is there a more... asynchronous way to handle this? 😬

Reply:
Probably only if Cromwell itself has a way of asynchronously handling backend initialization which, at first glance, looks to me to be... non-trivial.